keras/engine/training_arrays.py

```diff
@@ -180,7 +183,7 @@ def fit_loop(model, fit_function, fit_inputs,
             if shuffle == 'batch':
                 index_array = batch_shuffle(index_array, batch_size)
             elif shuffle:
-                np.random.shuffle(index_array)
+                rng.shuffle(index_array)
```

Comment:
The previous version already does not modify the global behavior of `np.random`. The purpose of this PR is not clear to me.